.travis.yml

```diff
@@ -4,7 +4,7 @@ services:
 - docker
 language: go
 go:
-- 1.8.1
+- 1.11.3
```

Comment:
This was lazy programming in an earlier version of my tool to just paste `runtime.Version()`. An in progress version of my tool will specify something like `"1.11.x"` which would be preferable as I think there was some go1.11.3 regression?